### PR TITLE
is210-week-03-warmup

### PR DESCRIPTION
**Pull request by ayine17**
Week 3 warmup- first python in this course

### CODE REVIEW

**Comment by ayine17** [comment]
@cheuschober i did not get comment or email about my pull. i don't whether it passed or failed

**Comment by cheuschober** [comment]
retest this please

**Comment by cheuschober** [comment]
retest this please

File: task_14.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+"""task_14"""
+
+
+IS_TRUE = True
+
+IS_FALSE = False
+
+IS_NONE = None
+
+INTEGER_EQUIV = (IS_TRUE == 1) & (IS_FALSE == 0)
```
Comment (cheuschober) [inline task_14.py]: @ayine17, very close here but you used the bitwise `&` (which passed the test) but not the *logical* `and`. There are notable differences in the operation of these two opearators.

**Pull request merged**